includes/wc-user-functions.php

```diff
@@ -108,7 +108,7 @@ function wc_create_new_customer( $email, $username = '', $password = '' ) {
 		$customer_id = wp_insert_user( $new_customer_data );
 
 		if ( is_wp_error( $customer_id ) ) {
-			return new WP_Error( 'registration-error', '<strong>' . __( 'ERROR', 'woocommerce' ) . '</strong>: ' . __( 'Couldn&#8217;t register you&hellip; please contact us if you continue to have problems.', 'woocommerce' ) );
+			return new WP_Error( 'registration-error', '<strong>' . __( 'Error', 'woocommerce' ) . ':</strong> ' . __( 'Couldn&#8217;t register you&hellip; please contact us if you continue to have problems.', 'woocommerce' ) );
```

Comment:
Well, let's keep this as `ERROR`, since WordPress does the same: https://github.com/WordPress/WordPress/blob/4a88d550545034c760bcae7adf992814a3ed6879/wp-admin/includes/user.php#L170